src/test/java/de/mediathekview/mserver/crawler/srf/parser/SrfSendungenOverviewJsonDeserializerTest.java

```diff
@@ -16,17 +17,20 @@
 
 @RunWith(Parameterized.class)
 public class SrfSendungenOverviewJsonDeserializerTest {
+  
+  private static final String MONTH_YEAR = LocalDateTime.now().getMonthValue() + "-" + LocalDateTime.now().getYear();
+  
   @Parameterized.Parameters
     public static Collection<Object[]> data() {
         return Arrays.asList(new Object[][] { 
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this array creation and simply pass the elements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3878)